pkg/cloudprovider/cloudprovider.go

```diff
@@ -101,7 +101,11 @@ func (c *CloudProvider) Create(ctx context.Context, machine *v1alpha5.Machine) (
 	instanceType, _ := lo.Find(instanceTypes, func(i *cloudprovider.InstanceType) bool {
 		return i.Name == instance.Type
 	})
-	return c.instanceToMachine(instance, instanceType), nil
+	outputMachine := c.instanceToMachine(instance, instanceType)
```

Comment:
```suggestion
	m := c.instanceToMachine(instance, instanceType)
```

nit: Maybe just consider shortening